core/tests/Test.Microsoft.Identity.SideBySide/UnifiedCacheTests.cs

```diff
@@ -540,7 +540,6 @@ private void AssertMsalCacheIsEmpty()
             Assert.IsTrue(msalCache.tokenCacheAccessor.GetAllAccessTokensAsString().Count == 0);
             Assert.IsTrue(msalCache.tokenCacheAccessor.GetAllRefreshTokensAsString().Count == 0);
             Assert.IsTrue(msalCache.tokenCacheAccessor.GetAllIdTokensAsString().Count == 0);
-            Assert.IsTrue(msalCache.tokenCacheAccessor.GetAllAccountsAsString().Count == 0);
```

Comment:
> Assert.IsTrue(msalCache.tokenCacheAccessor.GetAllAccountsAsString().Count == 0); [](start = 11, length = 81)

doesn't the cache initially come as empty? perhaps it will be 0 ? (though I also understand why you will not assert it... perhaps it can be asserted or it should be asserted that it's not 0 by injecting an account using a different client id) ?

Reply:
right, It is used for different purposes, reverted it back and added AssertNoCredentialsInMsalCache for validation of remove account api